src/transformer/descriptor/method/bodyReturnType.ts

```diff
@@ -1,23 +1,26 @@
 import * as ts from 'typescript';
 import { Scope } from '../../scope/scope';
 import { GetDescriptor } from '../descriptor';
-import { GetNullDescriptor } from '../null/null';
 
-export function GetReturnTypeFromBody(node: ts.ArrowFunction | ts.FunctionExpression | ts.MethodDeclaration | ts.FunctionDeclaration, scope: Scope): ts.Expression {
-  let returnValue: ts.Expression;
+export function GetReturnTypeFromBodyDescriptor(node: ts.ArrowFunction | ts.FunctionExpression | ts.MethodDeclaration | ts.FunctionDeclaration, scope: Scope): ts.Expression {
+  return GetDescriptor(GetReturnNodeFromBody(node), scope);
+}
+
+export function GetReturnNodeFromBody(node: ts.ArrowFunction | ts.FunctionExpression | ts.MethodDeclaration | ts.FunctionDeclaration): ts.Node {
+  let returnValue: ts.Node;
 
   const functionBody: ts.FunctionBody = node.body as ts.FunctionBody;
 
   if (functionBody.statements) {
     const returnStatement: ts.ReturnStatement = GetReturnStatement(functionBody);
 
     if (returnStatement) {
-      returnValue = GetDescriptor(returnStatement.expression, scope);
+      returnValue = returnStatement.expression;
     } else {
-      returnValue = GetNullDescriptor();
+      returnValue = ts.createNull();
```

Comment:
Should we use GetNullDescriptor or there is a special reason?  I understand that the wrapper GetNullDescriptor is not really useful but we would follow the same standard.

Reply:
done!